.github/workflows/release-docker-images.yml

```diff
@@ -108,19 +108,32 @@ jobs:
         run: |
           docker run --rm vegaprotocol/${{ matrix.app }}:local version || docker run --rm vegaprotocol/${{ matrix.app }}:local software version
 
-      - name: Build and push to Docker registries
-        id: docker_build
+      - name: Build and push to GitHub Container Registry
+        id: docker_build_github
```

Comment:
Maybe we can get rid of the github private registry as now everything is public

Reply:
We publish to DockerHub only tag releases: https://hub.docker.com/r/vegaprotocol/vega/tags.
We publish all `develop` commits to the GitHub registry.